diff -uNr lxc-0.7.5.orig//README.plamo lxc-0.7.5//README.plamo
--- lxc-0.7.5.orig//README.plamo	1970-01-01 09:00:00.000000000 +0900
+++ lxc-0.7.5//README.plamo	2011-11-08 14:50:14.310916651 +0900
@@ -0,0 +1,7 @@
+00-ptmx.rules は devpts を newinstance オプション付きでマウントした時，
+コンテナで起動した udev によって，ホストの /dev/ptmx の /dev/pts/ptmx 
+へのリンクが消去されて，新たに /dev/ptmx が作られてしまうのを防ぐため
+のルールです．newinstance オプション付きでマウントしない場合は不要です．
+
+devpts の newinstance オプションについてはカーネル付属文書の
+filesystems/devpts.txt を参照ください．
