--- ptexlive-20100711/7font-search.sh	2010-03-22 22:26:34.000000000 +0900
+++ build/7font-search.sh	2011-05-13 16:30:55.000000000 +0900
@@ -296,9 +296,9 @@
 echo "searching cmaps..."
 
 tarx cmap/mappingresources4pdf_2unicode.tar.xz $CMAP
-tarx $ISO_DIR/source/ghostscript-8.64.tar.xz $PTEXLIVE_TMP
-cmapinstall $PTEXLIVE_TMP/ghostscript-8.64/Resource/CMap
-$RM -r $PTEXLIVE_TMP || exit
+#tarx $ISO_DIR/source/ghostscript-8.64.tar.xz $PTEXLIVE_TMP
+#cmapinstall $PTEXLIVE_TMP/ghostscript-8.64/Resource/CMap
+#$RM -r $PTEXLIVE_TMP || exit
 
 echo ""
 echo "finishing..."
