diff -ur ptexlive-20100711/3make-install.sh build/3make-install.sh
--- ptexlive-20100711/3make-install.sh	2010-03-21 19:11:44.000000000 +0900
+++ build/3make-install.sh	2011-02-23 19:41:13.000000000 +0900
@@ -26,7 +26,6 @@
     --enable-multiplatform \
     --disable-omfonts \
     --with-ptexenc-enc=$KANJI_CODE \
-    --disable-all-pkgs \
     --enable-web2c \
     --disable-luatex \
     --disable-xetex \
diff -ur ptexlive-20100711/4extract-texmf.sh build/4extract-texmf.sh
--- ptexlive-20100711/4extract-texmf.sh	2010-03-07 19:22:24.000000000 +0900
+++ build/4extract-texmf.sh	2011-02-23 19:40:44.000000000 +0900
@@ -16,7 +16,6 @@
 tarcp  $TEXMF.install    $TEXMF     || exit
 $RM -r $TEXMF.install               || exit
 
-$LN $TEXLIVE_DIR/tlpkg      $TEXMF/.. || exit
 $LN $TEXLIVE_DIR/texmf-dist $TEXMF/.. || exit
 
 
