--- texlive-20091107-source.orig/texk/xdvik/Makefile.in	2009-06-16 21:01:38.000000000 +0900
+++ texlive-20091107-source/texk/xdvik/Makefile.in	2011-02-22 21:54:46.000000000 +0900
@@ -373,7 +373,7 @@
 common_includes = $(PS_DEF) $(KPATHSEA_INCLUDES) $(T1LIB_INCLUDES) $(x_cppflags)
 x_ldflags = $(X_LIBS) $(x_linker_options) $(iconv_libpath)
 x_extra_libs = $(X_EXTRA_LIBS) $(iconv_libs)
-xdvi_x_libs = $(x_ldflags) $(x_tool_libs) $(x_xmu_lib) -lXt \
+xdvi_x_libs = -L$(libdir) $(x_ldflags) $(x_tool_libs) $(x_xmu_lib) -lXt \
 	$(X_PRE_LIBS) $(x_ext_lib) $(x_xpm_libs) -lX11 $(x_extra_libs)
 common_ldadd = $(KPATHSEA_LIBS) $(T1LIB_LIBS) $(xdvi_x_libs)
 common_dependencies = $(KPATHSEA_DEPEND) $(T1LIB_DEPEND)
