--- chromium-55.0.2883.87/build/config/linux/BUILD.gn	2016-12-09 14:04:14.000000000 +0900
+++ build/build/config/linux/BUILD.gn	2017-01-11 15:15:34.052144405 +0900
@@ -13,6 +13,7 @@
 # is applied to all targets. It is here to separate out the logic that is
 # Linux-only. This is not applied to Android, but is applied to ChromeOS.
 config("compiler") {
+cflags_cc = [ "-fno-delete-null-pointer-checks" ]
   cflags = [ "-pthread" ]
   ldflags = [ "-pthread" ]
 }
